src/quacc/recipes/vasp/core.py

```diff
@@ -15,6 +22,104 @@
     from quacc.schemas._aliases.vasp import DoubleRelaxSchema, VaspSchema
     from quacc.utils.files import Filenames, SourceDirectory
 
+logger = logging.getLogger(__name__)
+
+
+@job
+def nscf_job(
```

Comment:
Thanks! This look good. The only logic we didn't end up including the NEDOS handling, but I think we can handle that later on if there's still interest.